_posts/2015-04-01-tackling-strongly-correlated-quantum-systems.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Tackling Strongly Correlated Quantum Systems on the Open Science Grid"
+date: 2015-04-01 12:00:00 -0600
+categories: Particle Physics
+---
+
+Duke University Associate Professor of Physics Shailesh Chandrasekharan and his graduate student Venkitesh Ayyar are
+using the Open Science Grid (OSG) to tackle notoriously difficult problems in quantum systems.
+
+<table>
+  <tbody>
+    <tr>
+      <td>
+        <div style="max-width: 160px">
```

Comment:
This HTML within each table cell seems excessively complex. And why does the image link to … itself?